### ISSUE TITLE
Android - Malformed screen output on PowerVR w/ OpenGL

### ISSUE THREAD

**Issue by SeongGino** [libretro/beetle-pce-libretro#41]
(Decided this was a core issue because this does not happen on virtually every other libretro core on said device)

As of current Beetle PCE and RetroArch 1.10.3, this is the output I'm finding on the device in question, running with `gl` video driver:
![Screenshot_20220421-195600](https://user-images.githubusercontent.com/7321839/164570206-f52d3e39-f4c5-4d95-804e-6fffc948cf57.png)

In case you couldn't tell, that's supposed to be _Bomberman._
However, it outputs properly on the same device using `vulkan`, as such:
![Screenshot_20220421-200122](https://user-images.githubusercontent.com/7321839/164570231-cf2bf4a5-15fa-402c-a57e-bad43564faf1.png)
(Both of these screenshots are standard Android screenies, to be representative of what the user sees).

This does not effect Beetle SuperGrafx on the same device.
I've also been unable to reproduce this on a OnePlus 7 Pro--both GL/Vulkan display this core properly on that device.

The Device in question is a Retroid Pocket 2+ (Plus, not original - big difference), which uses a Unisoc T310 - essentially a cut-down 1+3-core SD845, but with a PowerVR-GE8300.

**Comment by bslenul** [comment]
To the devs: note that this issue only seem to happen when `Settings > Video > Threaded Video` is disabled.

**Comment by vailkyte** [comment]
Looks related to #69

**Comment by bslenul** [comment]
Yep, it's displaying properly for me now! Thank you for the fix 👍
